capreolus/registry.py

```diff
@@ -173,8 +173,8 @@ def get_cache_path(self):
         """ Return a path encoding the module's config, which can be used for caching.
             The path is a function of the module's config and the configs of its dependencies.
         """
-
-        return CACHE_BASE_PATH / self.get_module_path(include_provided=True)
+        cache_base_path = Path(os.environ.get("CAPREOLUS_CACHE", os.path.expanduser("~/.capreolus/cache/")))
```

Comment:
So that calling `get_cache_path()` from the unit tests will look at the monkey patched env variable

Reply:
can we just monkeypatch CACHE_BASE_PATH and RESULTS_BASE_PATH in registry instead?